### PR TITLE
feat(react-query): more descriptions for cli status

### PR DESCRIPTION
**Pull request by gwansikk**
# Overview

I've added you as a co-author! @juno7803 
Thank you for your valuable feedback! Thanks to you, the explanation has become even better.

I have made the API display static for now. In the next version, I will make it dynamic to list and display the APIs. https://github.com/toss/suspensive/issues/1000#issuecomment-2197635287

| version 4 | version 5|
| --- | --- |
|image|image|

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: packages/react-query/src/scripts/utils/commands.ts [hunk 1]
```diff
@@ -3,6 +3,19 @@ import path from 'path'
 import { getPackageJson, getTanStackReactQueryPackageJson } from './package'
 import { switchVersion } from './switchVersion'
 
+export const version5APIs = ['SuspenseQuery', 'SuspenseQueries', 'SuspenseInfiniteQuery', 'QueryErrorBoundary']
```
Comment (gwansikk) [inline packages/react-query/src/scripts/utils/commands.ts:6]: I have temporarily declared the API list. In the next version, I will parse the TypeScript to list and display the APIs!

**Comment by manudeli** [comment]
@gwansikk changeset please

File: packages/react-query/src/scripts/utils/commands.ts (hunk 1 above)
Comment (manudeli) [reply packages/react-query/src/scripts/utils/commands.ts:6]: cool 👍 Fast resolve issue

**Review by manudeli — APPROVED** [review]
Much better 🚀🚀🚀🚀🚀

**Pull request merged**